Formula/gmic.rb

```diff
@@ -20,19 +20,26 @@ class Gmic < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "4e61637e97caf2fdd854f10c685ef304731a607a8a4e0161d49b0dcfeeac08ef"
   end
 
-  depends_on "cmake" => :build
```

Comment:
Did they get rid of the CMake build?

Reply:
At least from source tarball. We have had trouble updating to Makefile since v3.1.1 -- #101262

They do still have a copy in GitHub https://github.com/dtschump/gmic/blob/master/resources/CMakeLists.txt but not sure if that will continue to be maintained.